sympy/solvers/tests/test_solvers.py

```diff
@@ -2314,3 +2314,16 @@ def test_issue_20902():
     assert solve(f.subs({t: 3 * x + 3}).diff(x) > 0, x) == (S(-4)/3 < x) & (x < S(-2)/3)
     assert solve(f.subs({t: 3 * x + 4}).diff(x) > 0, x) == (S(-5)/3 < x) & (x < S(-1))
     assert solve(f.subs({t: 3 * x + 2}).diff(x) > 0, x) == (S(-1) < x) & (x < S(-1)/3)
+
+def test_issue_4886():
+    ra = Symbol('ra', positive=True)
+    rb = Symbol('rb', positive=True)
+    ra = R**2*a**2
+    rb = R**2*b**2
```

Comment:
We don't need to set `ra` to a symbol if we are just going to override that on the line below.

Reply:
@oscarbenjamin  Same for c1, c2, c3, ... ??